ggml.c

```diff
@@ -13990,6 +14126,7 @@ static thread_ret_t ggml_graph_compute_thread(void * data) {
     struct ggml_compute_state * state = (struct ggml_compute_state *) data;
 
     const int n_threads = state->shared->n_threads;
+    set_numa_thread_affinity(state->params.ith, n_threads);
```

Comment:
Should we do this only if NUMA ?

Reply:
The first line of the function returns if the system doesn't have at least two nodes, in which case it wouldn't have anything to do anyway because it would just be pinning all the threads to all the cores, which is the default.